lib/modules/onc_program/onc_standalone_public_launch_sequence.rb

```diff
@@ -0,0 +1,188 @@
+# frozen_string_literal: true
+
+require_relative './shared_onc_launch_tests'
+
+module Inferno
+  module Sequence
+    class OncStandalonePublicLaunchSequence < SequenceBase
+      include Inferno::Sequence::SharedONCLaunchTests
+
+      title 'Public Client Standalone Launch Sequence'
+
+      description 'Register Inferno as a public client with patient access and execute standalone launch.'
+
+      test_id_prefix 'OSLSP'
+
+      requires :onc_public_client_id,
+               :onc_public_scopes,
+               :onc_sl_oauth_authorize_endpoint,
```

Comment:
This doesn't display because there is only a field for `oauth_authorize_endpoint`.

I think you want to use `oauth_authorize_endpoint` here anyway for similar reasons as `oauth_token_endpoint`
![Screen Shot 2020-06-01 at 3 51 48 PM](https://user-images.githubusercontent.com/41651655/83448509-110a0700-a420-11ea-9bb8-81434f664b31.png)

Reply:
Also later in the test (line 129) `oauth_token_endpoint` is referenced, so it actually is the one that is required.

Reply:
Ah, yes, I failed to realize that the authorize endpoint was different than the token endpoint.  But unfortunately, it cannot be simply 'oauth_authorize_endpoint' because that will be prefilled by the EHR launches info, which I don't want.  Reworking now...